cmd/gke-gcloud-auth-plugin/cred.go

```diff
@@ -85,8 +85,10 @@ func newPlugin(tokenProvider tokenProvider) *plugin {
 var (
 	useApplicationDefaultCredentials = pflag.Bool("use_application_default_credentials", false, "Output is an ExecCredential filled with application default credentials.")
 	useEdgeCloud                     = pflag.Bool("use_edge_cloud", false, "Output is an ExecCredential for an Edge Cloud cluster.")
+	project                          = pflag.String("project", "", "Parent project of the Cluster.")
 	location                         = pflag.String("location", "", "Location of the Cluster.")
 	cluster                          = pflag.String("cluster", "", "Name of the Cluster.")
+	impersonateServiceAccount        = pflag.String("impersonate_service_account", "", "Impersonate a service account to retrieve tokens for the Cluster.")
```

Comment:
We should state that these flags are only applicable with --use_edge_cloud is enabled.